modules/gatk4/splitncigarreads/main.nf

```diff
@@ -8,7 +8,7 @@ process GATK4_SPLITNCIGARREADS {
         'quay.io/biocontainers/gatk4:4.2.5.0--hdfd78af_0' }"
 
     input:
-    tuple val(meta), path(bam)
+    tuple val(meta), path(bam), path(bai), path(intervals)
```

Comment:
Would not be better to pass `intervals` as a separate input channel?

Reply:
Could be an idea, but so far, everytime we passed intervals it's in the input channel containing the main input files.
I think it could be interesting to try out `each` at some point with intervals, but that would mean changing a lot of logic in multiples pipelines

Reply:
I tend to separate inputs since it seems to me that it usually saves doing an operator to join back output from other modules but fair enough :rocket: